_posts/en/newsletters/2019-11-27-newsletter.md

```diff
@@ -0,0 +1,280 @@
+    post][gibson wagners] that describes Wagner's algorithm in much more
+    detail with a mix of math, intuitive analysis, and topical
+    information Bitcoiners may find interesting (such as the amusing
+    tidbit of [Wagner's paper][] citing both Adam Back and Wei Dai
+    several years before Nakamoto did [the same][bitcoin.pdf], though
+    for different work).  Anyone interested in the developing their own
+    cryptographic protocols is recommended to read both posts, as each
+    compliments the other without being repetitive about the subject.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments of time to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:schmidty
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17265][] and [#17515][Bitcoin Core #17515] complete the
+  removal of the dependency on OpenSSL, which has been used since the
+  original Bitcoin 0.1 release but was the cause of [consensus
+  vulnerabilities][non-strict der], [remote memory leaks][heartbleed]
+  (potential private key leaks), [other bugs][cve-2014-3570], and [poor
+  performance][libsecp256k1 sig speedup].
+
+- [Bitcoin Core #16944][] updates the GUI to generate a [BIP174][]
+  Partially Signed Bitcoin Transaction (PSBT) and automatically copies
+  it to the clipboard if the user tries to create a transaction in a
+  watch-only wallet that has its private keys disabled.  The PSBT can
+  then be copied into another application for signing (e.g. [HWI][topic
+  hwi]).  The GUI doesn't yet provide a special dialog for copying the
+  signed PSBT back in for broadcasting.
+
+- [Bitcoin Core #17290][] changes which coin selection algorithm is used
+  in cases where the user requests certain inputs be used or asks that
+  the fee be selected from the payment amounts.  These now use Bitcoin
+  Core's normal default algorithm of Branch and Bound (BnB).  BnB was
+  designed to minimize fees and maximize privacy by optimizing for the
+  creation of changeless transactions.
+
+- [C-Lightning #3264][] includes several mitigations for [LND
+  #3728][], a bug in the implementation of gossip queries.  This
+  change also adds two new command line parameters useful for testing
+  and debugging, `--hex` and `--features`.
```

Comment:
Too much detail for readers, but for info:
`--hex` (boolean arg to print out messages in hex)
`--features=<hex>` (to send specified features in init)

Found this [code comment](https://github.com/ElementsProject/lightning/pull/3264/files#diff-a9e88a70de2a2100cf545f044575e4efR724) interesting :eyes: